jekyll/_cci2/language-ruby.md

```diff
@@ -7,60 +7,58 @@ categories: [language-guides]
 order: 8
 ---
 
-*[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
+*[Tutorials & 2.0 Sample Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
 
 This guide will help you get started with a Ruby on Rails application on CircleCI. 
 
 ## Overview
 
 If you’re in a rush, just copy the sample configuration below into a `.circleci/config.yml` in your project’s root directory and start building.
 
-Otherwise, we recommend reading our [walkthrough](#config-walkthrough) for a detailed explanation of our configuration.
-
-We maintain a reference Ruby on Rails project to show how to build Ruby on CircleCI 2.0:
+CircleCI maintains the sample Ruby on Rails project at the following links:
 
 - <a href="https://github.com/CircleCI-Public/circleci-demo-ruby-rails" target="_blank">Demo Ruby on Rails Project on GitHub</a>
 - <a href="https://circleci.com/gh/CircleCI-Public/circleci-demo-ruby-rails" target="_blank">Demo Ruby on Rails Project building on CircleCI</a>
 
 In the project you will find a commented CircleCI configuration file <a href="https://github.com/CircleCI-Public/circleci-demo-ruby-rails/blob/master/.circleci/config.yml" target="_blank">`.circleci/config.yml`</a>.
 
-We've built this application using the latest stable Rails version 5.1, `rspec-rails`, and [RspecJunitFormatter][rspec-junit-formatter] with PostgreSQL as our database of choice.
+The application uses the latest stable Rails version 5.1, `rspec-rails`, and [RspecJunitFormatter][rspec-junit-formatter] with PostgreSQL as the database.
 
-This application also assumes you're using one of our [CircleCI Images](http://circleci.com/docs/2.0/circleci-images).
+This application build also uses one of the pre-built [CircleCI Docker Images](http://circleci.com/docs/2.0/circleci-images).
 
 ## Pre-built CircleCI Docker images
 
 We recommend using a CircleCI pre-built image that comes pre-installed with tools that are useful in a CI environment. You can select the Ruby version you need from Docker Hub: <https://hub.docker.com/r/circleci/ruby/>.
 
-Database images for use as a secondary 'service' container are also available.
+Database images for use as a secondary 'service' container are also available on Docker Hub in the `circleci` directory.
 
 ---
 
 ## Sample Configuration
 
 {% raw %}
 ```YAML
-version: 2
-jobs:
-  build:
-    parallelism: 3
-    working_directory: ~/circleci-demo-ruby-rails
-    docker:
-      - image: circleci/ruby:2.4-node
-        environment:
+version: 2 # use CircleCI 2.0
+jobs: # a collection of steps
```

Comment:
Same comment as the [Django PR](https://github.com/circleci/circleci-docs/pull/2247#discussion_r181449857).